app/views/about/terms_and_conditions.en.html.erb

```diff
@@ -4,52 +4,62 @@
   <main class="govuk-main-wrapper" id="main-content">
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
-      
+
         <h1 class="govuk-heading-xl">Terms and conditions</h1>
         <p class="govuk-body">By using this service you agree to these terms and conditions.</p>
-        <p class="govuk-body">TODO: Add in appropriate content</p>
+
 
         <section class="moj-Section about-section">
-          <h2 class="govuk-heading-m">Responsibility for the service</h2>
+          <h2 class="govuk-heading-m">About the service</h2>
           <div class="Section__content govuk-govspeak gv-s-prose">
-            <p class="govuk-body">The Ministry of Justice (MOJ) runs this service.</p>
-            <p class="govuk-body">We try to make sure the information we provide is accurate and complete and the service is always available and virus free.</p>
-            <p class="govuk-body">The MoJ accepts no liability for:</p>
-            <ul class="govuk-list govuk-list--bullet">
-              <li>you being unable to access the service (for example due to high demand or because we've taken it down for maintenance)</li>
-              <li>indirect damages arising from you using the service or relying on its content</li>
-              <li>the content and availability of external websites that link to the service or which link from it</li>
-            </ul>
-          </div>
+            <p class="govuk-body">Check when a caution or conviction expires is a service provided by the Ministry of Justice (MoJ).</p>
```

Comment:
Let's use instead of the hardcoded service name, the helper (as we did in the cookies page for example) so if we ever change the name there is only one place that needs to be changed, `<%= service_name -%>`